src/app/app.module.ts

```diff
@@ -113,6 +113,9 @@ import { TosBannerComponent } from './tosBanner/tos-banner.component';
 import { ExporterStepComponent } from './workflow/snapshot-exporter-modal/exporter-step/exporter-step.component';
 import { SnaphotExporterModalComponent } from './workflow/snapshot-exporter-modal/snaphot-exporter-modal.component';
 import { ViewService } from './workflow/view/view.service';
+import { InfoTabService } from './workflow/info-tab/info-tab.service';
+import { WorkflowModule } from './shared/modules/workflow.module';
+import { SourceFileTabsService } from './source-file-tabs/source-file-tabs.service';
```

Comment:
Give than you removed the methods in InfoTabService and SourceFileTabs service, do you still need these 3 imports and their declarations below?